site/content/getting-started/_index.md

```diff
@@ -226,10 +173,7 @@ Verify web access by browsing to [127.0.0.1](http://127.0.0.1). You can refresh
 
 Congratulations, you have installed Contour with a backend web application workload using Helm .
 
-Note: When you are done with the cluster, you can delete it by running:  
-```yaml
-$ kind delete cluster
-```  
+
```

Comment:
When we have the user access the app, we should account for both a local (KinD) cluster (so using 127.0.0.1) as well as a remote cluster (so need the LB address, or to port-forward a NodePort). Would need to add some instructions around each of those cases.